quickstart/WORKSPACE

```diff
@@ -23,9 +23,9 @@ load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 # NOTE: Update this version and SHA256 as needed.
 http_archive(
     name = "com_github_googleapis_google_cloud_cpp_spanner",
-    sha256 = "17f2d5f76d41987114e6eb726ca8950a705911e9bacb6bf3264b70ae359d4c9b",
-    strip_prefix = "google-cloud-cpp-spanner-0.9.0",
-    url = "https://github.com/googleapis/google-cloud-cpp-spanner/archive/v0.9.0.tar.gz",
+    sha256 = "9d5b33837d1164c3abb8a7c5f6d394490e4c3fa3b0a27fa7496cc0eea93508c7",
+    strip_prefix = "google-cloud-cpp-spanner-1.0.0",
+    url = "https://github.com/googleapis/google-cloud-cpp-spanner/archive/v1.0.0.tar.gz",
```

Comment:
Should this be part of the GA release?

Reply:
Cannot be, you cannot create the SHA until after the release is cut.

Reply:
And of course, this  embedded SHA number is part of the release's SHA, so unless we have a good way to find fixed points in the SHA function, we (I think) cannot.